smart-table-module/js/Table.js

```diff
@@ -176,11 +185,15 @@
 
             /**
              * insert a new column in scope.collection at index or push at the end if no index
+             * call setDefaultSortColumn to set lastCoulmnSort to already sorted column
              * @param columnConfig column configuration used to instantiate the new Column
              * @param index where to insert the column (at the end if not specified)
              */
             this.insertColumn = function (columnConfig, index) {
                 var column = new Column(columnConfig);
+                if (angular.isDefined(column.reverse)) {
```

Comment:
So, if we provide "reverse: true" in the configuration of a column, that column will be sorted in ascending order by default when the table loads? Can we add this to the README, just so future developers don't need to dig into the code to know this?